### ISSUE TITLE
bibliography preview via right-click seldom works

### ISSUE THREAD

**Issue by summerjob** [ahrm/sioyek#1085]
** Environment:**
macOS 12

**Issue:**
Previewing the bibliography entry via right-click on in-text citation seldom works if citation is not pre-linked in the PDF. Is there any way to improve detection in settings or by updating any dependencies?

**Comment by ahrm** [comment]
Can you share the document where it does not work? Maybe it is using a different citation style from the documents I have tried.

**Comment by ahrm** [comment]
Yeah we mainly work with bracket reference style (e.g. [12]). However, in development branch we have a feature where you can select a reference and right click it to preview its location, that feature does work with the citation style of this document.
